src/main/java/emu/grasscutter/command/commands/JoinCommand.java

```diff
@@ -13,7 +13,7 @@
 import static emu.grasscutter.utils.Language.translate;
 
 @Command(label = "join", usage = "join [AvatarIDs] such as\"join 10000038 10000039\"",
-        description = "commands.join.description", permission = "player.join")
+        description = "commands.join.description", permission = "server.join")
```

Comment:
I forgot to ask this, but can you also add a `permissionTargeted` so other players without the right permission cant force join/remove characters you dont want into your party?

Reply:
OK. I will hand on it.

Reply:
> I forgot to ask this, but can you also add a `permissionTargeted` so other players without the right permission cant force join/remove characters you dont want into your party?

fixed

Reply:
By the way, Is there any division standard between server permission and player permission? How can I choice?
player permission———client only?
server permission———both side?
But why
![image](https://user-images.githubusercontent.com/73888354/168524484-a9603c9a-7af9-4b36-94ec-a3b919066a76.png)
![image](https://user-images.githubusercontent.com/73888354/168524506-4241c561-ca92-4db1-b7de-22f9c6f65b3e.png)

Reply:
telepory can be excute on the both side.
![image](https://user-images.githubusercontent.com/73888354/168524593-0fb78bd8-1f48-475e-9579-3ada2532118a.png)